app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,67 @@
+class OrdersController < ApplicationController
+ 
+  before_action :create_order, only: [:show, :edit, :update, :destroy]
+  def index
+    @orders = Order.all
+    if params[:search]
+      if params[:search] == "booked"
+        @orders = Order.where(status: "booked")
+      elsif params[:search] == "cancelled"
+        @orders = Order.where(status: "cancelled")
+      else
+        @orders = Order.all
+      end
+    end
+  end
+
+  def show
+    @order = Order.find_by_id(params[:id])
```

Comment:
Don't needs to add this: @order = Order.find_by_id(params[:id])
As you are already finding order in before action method.